test/serializer/abstract/DebugValue.js

```diff
@@ -0,0 +1,9 @@
+let x = global.__abstract ? __abstract("boolean", "true") : true;
```

Comment:
Make this test so that it can fail. For example, assert that "debugger" is present in the output.

Reply:
That's a good point. We might need to add `debugger` to the eslint check too.

Reply:
There is no `debugger` in the output (as you seem to have observed in your comment about the empty_statement op).

Reply:
That's what I mean, we need to ensure there is none. We also need to be careful of residual functions that might not be be prepacked (thus the possible eslint rule change) containing the `__debugValue`.

Reply:
@trueadm I thought about adding `does not contain: debugger` when I checked in the test, but decided to leave it out because there's no path for it to get there. I added the test as an illustration of how to use this helper, and for test coverage. But I'm happy to add the line - it helps with documentation, as well.